fdbserver/TLogServer.actor.cpp

```diff
@@ -973,9 +980,10 @@ ACTOR Future<Void> updatePersistentData( TLogData* self, Reference<LogData> logD
 	return Void();
 }
 
-// This function (and updatePersistentData, which is called by this function) run at a low priority and can soak up all CPU resources.
-// For this reason, they employ aggressive use of yields to avoid causing slow tasks that could introduce latencies for more important
-// work (e.g. commits).
+// This function (and updatePersistentData, which is called by this function) run at a low priority and can soak up all
+// CPU resources. For this reason, they employ aggressive use of yields to avoid causing slow tasks that could introduce
+// latencies for more important work (e.g. commits).
+// Q: Can someone explain what this actor is doing?
 ACTOR Future<Void> updateStorage( TLogData* self ) {
```

Comment:
@sfc-gh-almiller  It seems this actor is making tLog's data from `logData->persistentDataVersion` to `logData->version` durable on disk?

Reply:
This actor is just a loop that calls updatePersistentData and popDiskQueue whenever (a) there's data to be spilled or (b) we should update metadata after some commits have been fully popped.